setup.py

```diff
@@ -21,6 +21,9 @@ def _find_version(file_path):
     raise RuntimeError('Unable to find version string.')
 
 
+CLI_DEPS = ['click>=7.0.0', 'tabulate']
```

Comment:
If possible it may be better to avoid trying to be smart there by using variables:
1. setup.py is replaced in the declarative setup.cfg and this would no longer be supported then
2. Tools like scancode-toolkit ;) would not be having their best hair day when trying to interpret these statically https://github.com/nexB/scancode-toolkit/blob/develop/src/packagedcode/pypi_setup_py.py 
I would suggest instead to inline  and repeat these `'click>=7.0.0', 'tabulate'` in each of cli and full requires.

Reply:
I've removed the variable. I'm going to change the packaging system to `setup.cfg` hopefully before the end of the month.